README.md

```diff
@@ -90,12 +90,44 @@ To select a project/solution:
 - At the top of that pane, select the prjoect you want to debug from the dropdown
 - Click the `Run` &#9658; button or press `F5` to start debugging
 
-### To run unit/integration tests
+### To run unit/integration tests from the GUI
+
+- Setup: Add Cosmos Key to secrets and copy to local tmp directory
+
+  ```bash
+    # add cosmos key to secrets
+    echo $(eval az cosmosdb keys list -n ngsa-asb-dev-cosmos -g rg-ngsa-asb-dev-cosmos --query primaryMasterKey -o tsv) > src/LodeRunner/secrets/CosmosKey
+
```

Comment:
We should be using the "test" CosmosDB for integration tests.  The name is [ngsa-asb-test-cosmos](https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/648dcb5a-de1e-48b2-af6b-fe6ef28d355c/resourceGroups/rg-ngsa-asb-test-cosmos/providers/Microsoft.DocumentDB/databaseAccounts/ngsa-asb-test-cosmos)